src/ideations/ideations.controller.ts

```diff
@@ -80,10 +136,30 @@ export class IdeationsController {
         summary:
             "Updates an ideation given a ideationId (int) and the that user that created it is logged in.",
     })
+    @ApiResponse({
+        status: HttpStatus.UNAUTHORIZED,
+        description:
+            "Invalid uuid or teamID. User is not authorized to preform this action.",
+        type: UnauthorizedErrorResponse,
+    })
+    @ApiResponse({
+        status: HttpStatus.NOT_FOUND,
+        description: "Ideation with given ID does not exist.",
+        type: NotFoundErrorResponse,
+    })
+    @ApiResponse({
```

Comment:
Can you check this one? The conflict error code is 409 but the type used for the example is 404.
<img width="640" alt="Screenshot 2024-01-12 at 12 25 47 PM" src="https://github.com/chingu-x/chingu-dashboard-be/assets/80556079/99401977-acd6-4e24-9a68-b868d6ca597a">